src/components/views/messages/MessageActionBar.tsx

```diff
@@ -135,15 +135,20 @@ interface IMessageActionBarProps {
     getTile: () => any | null;
     getReplyThread: () => ReplyThread | undefined;
     permalinkCreator?: RoomPermalinkCreator;
-    onFocusChange: (menuDisplayed: boolean) => void;
-    isQuoteExpanded?: boolean;
+    onFocusChange?: (menuDisplayed: boolean) => void;
     toggleThreadExpanded: () => void;
+    isInThreadTimeline?: boolean;
```

Comment:
Can I suggest to make it an enum instead, so that if we ever want to extend it more in the future we won't have to add more props, or refactor this one? Had similar problems already and adding a rendering context enum seems like a good enough solution for now.